webapp/src/components/chat/ChatInput.tsx

```diff
@@ -177,6 +183,7 @@ export const ChatInput: React.FC<ChatInputProps> = ({ isDraggingOver, onDragLeav
             <Alerts />
             <div className={classes.content}>
                 <Textarea
+                    ref={textAreaRef}
```

Comment:
Why not just 
```suggestion
                    <Textarea
                         autoFocus
```

Reply:
it doesn't work when the chat is switched since it only autofocuses the input when the page [first loads](https://www.w3schools.com/jsref/prop_text_autofocus.asp).